src/sentry/integrations/slack/action_endpoint.py

```diff
@@ -39,6 +39,13 @@ class SlackActionEndpoint(Endpoint):
     authentication_classes = ()
     permission_classes = ()
 
+    def api_error(self, error):
+        return self.respond({
+            'response_type': 'ephemeral',
+            'replace_original': False,
+            'text': u'Action failed: {}'.format(error.body['detail']),
```

Comment:
Are you sure this key actually exists in the body? I don't see it in the stacktrace of SENTRY-690. Or is there some layer of indirection that I'm missing?

Reply:
Nah you're right, this probably can't generically expect for `detail` to exist. I'm just going to change the message to 'You cannot perform this action'. The particular error here was actually fixed in https://github.com/getsentry/sentry/pull/7953